src/App.js

```diff
@@ -9,30 +9,48 @@ import CalculationState from "./context/calculation/CalculationState";
 
 // import './App.css';
 
-const App = () => {
-  return (
-    <TransactionState>
-      <CalculationState>
-        <Fragment>
-          <h1 className="center-align">Crypto Capital Gains Calculator</h1>
-          <div className="container">
-            <p className="flow-text center">
-              This app was built mainly to get practice designing and building
-              React apps. I am not an accountant and make no guarantees that the
-              calculations performed within this app are accurate. I encourage
-              you to seek the help of an accountant or tax expert with any
-              questions regarding tax obligations for crypto transactions.
-            </p>
-          </div>
-          <div className="divider"></div>
-          <TransactionForm />
-          <TransactionTable />
-          <ErrorMessage />
-          <CalculationTable />
-        </Fragment>
-      </CalculationState>
-    </TransactionState>
-  );
-};
+class App extends Component {
+  state = {
```

Comment:
should this go into a `constructor()`?

Reply:
https://reactjs.org/docs/react-component.html#constructor
>You should not call setState() in the constructor(). Instead, if your component needs to use local state, assign the initial state to this.state directly in the constructor

> Constructor is the only place where you should assign this.state directly. In all other methods, you need to use this.setState() instead.

Reply:
https://www.robinwieruch.de/react-state-without-constructor
> By using an alternative class syntax, you can leave out the constructor and initialize the state as class field declaration. However, you don't have access to the props anymore.

> The syntax is not widely adopted yet, because class field declaration are a new feature in JavaScript, but once they will be supported by all browsers, they may be used more widely in React to create initial state in a class component.

Reply:
sounds like mixed opinions on this. let's go with `constructor()` for now since that's the official documentation

Reply:
Done